extensions/src/main/java/org/apache/falcon/extensions/jdbc/ExtensionMetaStore.java

```diff
@@ -154,7 +157,7 @@ public void storeExtensionJob(String jobName, String extensionName, List<String>
         EntityManager entityManager = getEntityManager();
         try {
             beginTransaction(entityManager);
-            entityManager.persist(extensionJobsBean);
+            entityManager.merge(extensionJobsBean);
```

Comment:
https://github.com/apache/falcon/pull/328/files#diff-b03505aec96486daeb519e68aba63d2dR445

While submitting an extension job, its process and feeds are submit in a sequential manner, so I have made this merge to support update of process and feed list.